### ISSUE TITLE
Contact Edit fails: 'Could not find contact with id 3'

### ISSUE THREAD

**Issue by lkallevig** [JeremyLikness/BlazorWasmEFCoreExample#7]
View and Delete work as expected. 

```<p>Could not find contact with id @ContactId.</p>``` is the error message. That extra . looks like a problem but removing it changes nothing.

Did I break something after download?
Kal

**Comment by JeremyLikness** [comment]
I just tested and it worked fine. Is this clicked "edit" from the grid? With logging turned on, do you see any generated errors or messages?

**Comment by lkallevig** [comment]
Yes, it is clicked ‘edit’ from the grid.

I just grabbed a fresh clone from the site and did the migration step which linked it to the existing database. It works as expected.

I did a comparison of the two versions and some of the code has changed. Also, yesterday I updated all the Nugget links to the latest which I have not done with the today clone. In any case, seems to work now.

Thanks for your help,
Kal
